crates/wasm-encoder/src/tags.rs

```diff
@@ -0,0 +1,73 @@
+use super::*;
+use std::convert::TryFrom;
+
+/// An encoder for the tag section.
+#[derive(Clone, Debug)]
+pub struct TagSection {
```

Comment:
Can you add an example here? All of the other sections have examples at the top (e.g. https://github.com/bytecodealliance/wasm-tools/blob/main/crates/wasm-encoder/src/globals.rs#L5-L23) which is nice for people using this crate.